data_safe_haven/serialisers/yaml_serialisable_model.py

```diff
@@ -1,5 +1,6 @@
 """A pydantic BaseModel that can be serialised to and from YAML"""
 
+from difflib import unified_diff
```

Comment:
Is this part of the standard library or do we need to add a package?

Reply:
stdlib